src/makeCalculator.js

```diff
@@ -4,7 +4,27 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    add: (n) => (calc.result += n),
+    subtract: (n) => (calc.result -= n),
+    multiply: (n) => (calc.result *= n),
+    divide: (n) => (calc.result /= n),
```

Comment:
It is necessary to add a check for 0, because 0 can be divided by according to the rules of mathematics